CMakeLists.txt

```diff
@@ -67,14 +68,23 @@ if(BUILD_TOOLS)
         set_target_properties(gltfpack PROPERTIES INSTALL_RPATH ${RPATH})
     endif()
 
-    if(EXISTS glfw)
+    find_package(glfw3 3.3 QUIET)
+
+    if (glfw3_FOUND)
+        message(STATUS "Found GLFW3 via find_package()")
+        add_executable(lodviewer tools/lodviewer.cpp tools/meshloader.cpp)
+        target_link_libraries(lodviewer ${GLFW3_LIBRARY} meshoptimizer)
```

Comment:
If the code below declared GLFW3_LIBRARY variable locally, these two lines (that will change whenever the set of source files required changes) could be moved to be outside of GLFW discovery code.